app/views/hiring_staff/vacancies/application_details/new.html.haml

```diff
@@ -16,6 +16,12 @@
                 as: :email,
                 required: true
 
+      = f.input :application_link,
```

Comment:
I feel like a hint would work well here as I can imagine `Application URL` will not be clearly understood by all.

Reply:
fixed, it was mapped to the wrong locale as I copy pasted the gds fields below and edited 🤔